starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -2499,35 +2515,26 @@ sub time2date {
 }
 
 ###################################################################################
-sub count_good_stars{
+sub count_guide_stars{
 ###################################################################################
     my $self=shift;
     my $c;
-    my $clean_acq_count = 0;
-    my $clean_gui_count = 0;
-    $self->{count_nowarn_stars}{ACQ} = $clean_acq_count;
-    $self->{count_nowarn_stars}{GUI} = $clean_gui_count;
+    my $gui_count = 0.0;
 
-    return unless ($c = find_command($self, 'MP_STARCAT'));
+    return 0.0 unless ($c = find_command($self, 'MP_STARCAT'));
     for my $i (1 .. 16){
-        my $type = $c->{"TYPE$i"};
-        next if ($type eq 'NUL');
-        next if ($type eq 'FID');
-	if ($type =~ /ACQ|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_acq_count++;
-	    }
-	}
-	if ($type =~ /GUI|BOT/){
-	    unless ($self->check_idx_warn($i)){
-		$clean_gui_count++;
-	    }
+	if ($c->{"TYPE$i"} =~ /GUI|BOT/){
+            my $mag = $c->{"GS_MAG$i"};
+            # Compute fractional guide star count using magnitude bins and fractions
+            # defined in ORViewer. (Note tab-ternary is if/elsif/else)
+            my $star_contrib = $mag <= 10.0  ? 1.0
+                             : $mag <= 10.2  ? 0.5
```

Comment:
Shouldn't this be 0.75 then 0.5?

Reply:
That would make more sense wouldn't it.  Good catch.